.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,18 @@
+name: Run action - pull request
+on: [pull_request]
+
+# You may not be able to test a pull request using run_action.yml 
+# because of the absolute path to the docker image.
+
+# So this is the replacement.
```

Comment:
```suggestion
# It is not possible to test a pull request using run_action.yml 
# because the action uses the deployed docker image from GHCR.

# So this is the next best thing.
```